src/main/resources/edu/cornell/kfs/module/purap/document/workflow/VendorCreditMemoDocument.xml

```diff
@@ -40,7 +40,8 @@
                         </branch>
                         <join name="JoinRequiresImageAttachment"/>
                     </split>
-                    <role name="Account" nextNode="TreasuryManager" nextAppDocStatus="Awaiting Treasury Manager Approval"/>
+                    <role name="Account" nextNode="ObjectCode"/>

```

Comment:
Does this route node need a nextAppDocStatus attribute value to indicate it is "Awaiting Object Code Review"?

Reply:
Vendor Credit Memo documents did not receive a new app doc status related to Object Code routing, so we don't need to add a "nextAppDocStatus" setting for it.

Reply:
Thanks for clarifying.